### PR TITLE
Fix copy/paste error on SME description.

### LINKED ISSUE #1: SME description is wrong
The description for a SME is wrong here: https://github.com/PagerDuty/incident-response-docs/blob/master/docs/before/different_roles.md#what-is-it-3

It looks to be almost a copy/paste of the "Scribe > How can I become one?" section.

### PR DESCRIPTION
**Pull request by richadams**
Oops, I had a bit of a copy/paste error here. This updates it to the correct description.

Resolves #1

### CODE REVIEW

File: docs/before/different_roles.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Follow our [Scribe training guide](/training/scribe.md), and then notify the Inc
 ## Subject Matter Expert
 
 ### What is it?
-Follow our Scribe training guide, and then notify the Incident Commanders in #incident-commanders that you would like to be considered for scribing for the next incident.
+A Subject Matter Expert (SME), sometimes called a "Resolver", is a domain expert or designated owner of a component or service that is part of the PagerDuty software stack.
```
Comment (esigler) [inline docs/before/different_roles.md]: `PagerDuty software stack` -> just the generic `software stack` perhaps?  Otherwise LGTM.

Comment (richadams) [inline docs/before/different_roles.md]: Will leave for now so it matches internal docs. I need to do a pass at some point to "genericise" it anyway (is that a word?... it is now)

Comment (Zaram22) [reply docs/before/different_roles.md:94]: > `PagerDuty software stack` -> just the generic `software stack` perhaps?  Otherwise LGTM.

Comment (Zaram22) [reply docs/before/different_roles.md:94]: > Will leave for now so it matches internal docs. I need to do a pass at some point to "genericise" it anyway (is that a word?... it is now)

**Pull request merged**